source/instructors/setupcourses/assignment-settings/exam-proctoring.rst

```diff
@@ -46,4 +46,4 @@ The Exam Proctoring settings are used to enable various options for assignment e
 
   On the **Students** tab in the course, click the menu icon (3 horizonatal dots) and choose **Reset Single Login**.
 
-
+- **Authentication** - If enabled, your students will need to authenticate before they can open the assignment. This option can only be used if you have configured :ref:`SSO Integration <sso-integration>` for the organisation.
```

Comment:
Line 49: organisation -> organization

Can we add a word or two here to explain what this authentication means - how does it support the exam process - I truly am unclear on this.

Reply:
Far as I read it all, with this enabled no matter how they logged into Codio (directly, through LMS or through SSO provider) they have to authenticate again before starting the assignment (ie enter email/pwd etc)